app/code/core/Mage/Api/Model/Server/Handler/Abstract.php

```diff
@@ -33,11 +33,12 @@ public function __construct()
      * @param int $errorCode
      * @param string $errorMessage
      * @param string $errorFile
+     * @param int $errLine
      * @return bool
      */
     public function handlePhpError($errorCode, $errorMessage, $errorFile, $errLine)
     {
-        Mage::log($errorMessage . ' in ' . $errorFile . ' on line ' . $errLine, Zend_Log::ERR);
+        Mage::log($errorMessage . '  in ' . $errorFile . ' on line ' . $errLine, Zend_Log::ERR);
```

Comment:
```suggestion
        Mage::log($errorMessage . ' in ' . $errorFile . ' on line ' . $errLine, Zend_Log::ERR);
```

Reply:
There is also a double space in app/code/core/Mage/Core/functions.php.